src/pages/en/reference/cli-reference.md

```diff
@@ -2,41 +2,106 @@
+
+Use the CLI by running one of the **commands** documented on this page, optionally followed by any **flags**. Flags customize the behavior of a command. For example, to start the development server on port `8080`, you would combine the `astro dev` command with the `--port` flag: `astro dev --port 8080`.
+
+In most cases you will use the CLI via your package manager:
+
+<PackageManagerTabs>
+  <Fragment slot="npm">
+  ```shell
+  npx astro dev --port 8080
+  ```
+  </Fragment>
+  <Fragment slot="pnpm">
+  ```shell
+  pnpm astro dev --port 8080
+  ```
+  </Fragment>
+  <Fragment slot="yarn">
+  ```shell
+  yarn astro dev --port 8080
+  ```
+  </Fragment>
+</PackageManagerTabs>
+
+If you started your project using [the `create astro` wizard](/en/install/auto/#1-run-the-setup-wizard), you can also use the scripts in `package.json` for a shorter version of these commands. See the `README.md` in your project for details of which commands are available.
+
+<PackageManagerTabs>
+  <Fragment slot="npm">
+  ```shell
+  # run the dev server on port 8080 using the `start` script in `package.json`
+  npm start -- --port 8080
+  ```
+  (The extra `--` before the `--port` flag is necessary for `npm` to pass your flags to the `astro` command.)
+  </Fragment>
+  <Fragment slot="pnpm">
+  ```shell
+  # run the dev server on port 8080 using the `start` script in `package.json`
+  pnpm start --port 8080
+  ```
+  </Fragment>
+  <Fragment slot="yarn">
+  ```shell
+  # run the dev server on port 8080 using the `start` script in `package.json`
+  yarn start --port 8080
+  ```
+  </Fragment>
+</PackageManagerTabs>
+
+## `astro dev`
+
+Runs  Astro's development server. It starts an HTTP server which responds to requests for routes or pages that are specified within `src/pages` directory (unless overridden by your `pages` option set in the project [configuration](/en/reference/configuration-reference/)).
+
+<h3>Flags</h3>
+
+Use these flags to customize the behavior of the Astro dev server. For flags shared with other Astro commands, see [common flags](#common-flags) below.
+
+#### `--port <number>`
 
-### `astro dev`
+Specifies which port to run on. Defaults to `3000`.
 
-Runs  Astro's `dev` server. It starts an HTTP server which responds to requests for routes or pages that are specified within `src/pages` directory (unless overridden by your `pages` option set in the project [configuration](/en/reference/configuration-reference/)).
+#### `--host [optional host address]`
 
-**Flags**
+Sets which network IP addresses the dev server should listen on (i.e. non-localhost IPs). This can be useful for testing your project on local devices like a mobile phone during development.
 
-#### `--port`
+- `--host` — listen on all addresses, including LAN and public addresses
+- `--host [custom-address]` — expose on a network IP address at `[custom-address]`
```

Comment:
Should this be `<custom-address>`, or is the idea that angle brackets are for less specific types

Reply:
I _think_ the convention is `[]` for optional parts of the command, `<>` for required, but I’m not wedded to it, as who knows that??

Reply:
Also, in this context we’ve just shown the version _without_  an address so I think I’ll go with your suggestion 🙌 

```suggestion
- `--host <custom-address>` — expose on a network IP address at `<custom-address>`
```